css/_variables.scss

```diff
@@ -1,4 +1,9 @@
 /**
+* Theme
+*/
+@import 'themes/main';
```

Comment:
We're importing this one in main.scss as well, so I'm not sure why we're importing it again here.